src/components/specific/ConfigureModal/ConfigureModal.js

```diff
@@ -0,0 +1,65 @@
+import { useDispatch, useSelector } from "react-redux";
+import { Modal } from "../../common/Modal";
+import { Button } from "../../common/Button";
+import { Select } from "../../common/Select";
+import "../../../assets/fonts/font-poppins.css";
+import "./ConfigureModal.css";
+
+export function ConfigureModal({
+    closeModal,
+}) {
+    const dispatch = useDispatch();
+    const clockPrecision = useSelector(state => state.clockPrecision);
+    const clockType = useSelector(state => state.clockType);
+
+    const modalSubmitHandler = event => {
+        event.preventDefault();
+        dispatch({
+            type: "clockPrecision",
+            value: event?.target?.clockprecision?.value
+        });
+        closeModal();
+    }
+
+    return(
+        <Modal
+            onModalCloseHandler={ closeModal }
+            content={
+                <div className="padding-20">
+                    <form className="configure-form" onSubmit={ modalSubmitHandler }>
+                        <h3 className="poppins-regular font-color-light">Digital Clock Precision</h3>
+                        <Select
+                            options={[
```

Comment:
import the select options from a different file

Reply:
Fixed in commit "Store data in separate configurations file".